mobile/src/calendar/event-dialog/event-dialog-base.tsx

```diff
@@ -69,9 +69,18 @@ export class EventDialogBaseImpl extends Component<EventDialogBaseProps> {
 
         return (
             <View style={layout.buttons}>
-                <CalendarActionButton title={cancelLabel} onPress={this.cancel} style={buttons.cancel as ViewStyle} textStyle={buttons.cancelLabel as TextStyle} disabled={this.props.inProgress} />
-                <CalendarActionButton title={acceptLabel} onPress={this.accept} style={buttons.accept as ViewStyle} textStyle={buttons.acceptLabel as TextStyle} disabled={disableAccept || this.props.inProgress} />
-                {this.props.inProgress && <ActivityIndicator size={'small'} style={buttons.progressIndicator} />}
+                {
+                    !!cancelLabel &&
+                    <CalendarActionButton title={cancelLabel} onPress={this.cancel}
+                                          style={buttons.cancel as ViewStyle}
+                                          textStyle={buttons.cancelLabel as TextStyle}
+                                          disabled={this.props.inProgress}/>
+                }
+                <CalendarActionButton title={acceptLabel} onPress={this.accept}
+                                      style={buttons.accept as ViewStyle}
+                                      textStyle={buttons.acceptLabel as TextStyle}
+                                      disabled={disableAccept || this.props.inProgress}/>
+                {this.props.inProgress && <ActivityIndicator size={'small'} style={buttons.progressIndicator}/>}
```

Comment:
I would kill for such things:
`{this.props.inProgress && <ActivityIndicator size={'small'} style={buttons.progressIndicator}/>}`